release-content/0.16/migration-guides/_guides.toml

```diff
@@ -1030,3 +1030,9 @@ title = "Use `target_abi = \"sim\"` instead of `ios_simulator` feature"
 prs = [17702]
 areas = []
 file_name = "17702_Use_target_abi__sim_instead_of_ios_simulator_feature.md"
+
+[[guides]]
```

Comment:
Can you move this entire section to be next to other ECS-related migrations? They are around line 200